rules/allied-naval.yaml

```diff
@@ -129,3 +129,85 @@ aegis:
 	Voiced:
 		VoiceSet: AlliedNavalVoice
 
+dlph:
+	Inherits@1: ^ExistsInWorld
+	Inherits@2: ^SpriteActor
+	Inherits@3: ^GainsExperience
+	Buildable:
+		Queue: Ship
+		BuildPaletteOrder: 30
+		Prerequisites: gatech, ~gayard
+	Valued:
+		Cost: 500
+	Tooltip:
+		Name: Dolphin
+		Description: Trained dolphin\narmed with sonic beams.\n  Strong vs Ships
+	Armor:
+		Type: Light
+	Health:
+		HP: 200
+	Mobile:
+		ROT: 6
+		Speed: 90
+		TerrainSpeeds:
+			Water: 100
+		Voice: Move
+	SelectionDecorations:
+		Palette: pips
+	Selectable:
+	Voiced:
+		VoiceSet: DolphinVoice
+	Armament:
+		Weapon: SonicZap
+	AttackFrontal:
+		Voice: Attack
+	AutoTarget:
+	RevealsShroud:
+		Range: 4c0
+	Cloak:
+		CloakTypes: Underwater
+		InitialDelay: 50
+		CloakDelay: 75
+		CloakSound: vnavupa.wav
+		UncloakSound: vnavupa.wav
+		WhileCloakedUpgrades: underwater
+		UncloakOnAttack: false
+	Targetable:
+		TargetTypes: Ground, Water, Repair
+		UpgradeTypes: underwater
+		UpgradeMaxEnabledLevel: 0
+	Targetable@underwater:
+		TargetTypes: Underwater, Repair
+		UpgradeTypes: underwater
+		UpgradeMinEnabledLevel: 1
+		IsPlayerPalette: true
+	DetectCloaked:
+		CloakTypes: Underwater
+		Range: 4c0
+	HiddenUnderFog:
+	ActorLostNotification:
+	Guard:
+		Voice: Attack
+	Guardable:
+	DeathSounds@normal:
+		DeathTypes: ExplosionDeath, BulletDeath, FlameDeath, ElectroDeath, RadiationDeath
```

Comment:
You shouldn't need to define all that. If `DeathTypes` is empty a sound will always play: https://github.com/OpenRA/OpenRA/pull/10156